packages/web/src/components/Home/NavBar/Leaderboard.jsx

```diff
@@ -149,6 +153,167 @@ export default function Leaderboard({
+                      +3🍪
+                    </Text>
+                  </Flex>
+
+                  {/* Cookies for resolving a found Item */}
+                  <Flex
+                    bg={"white"}
+                    boxShadow={"7px 7px 14px grey"}
+                    borderRadius={"xl"}
+                    p={5}
+                    gap={2}
+                    alignItems={"center"}
+                    justifyContent={"center"}
+                    flexDir={"column"}>
+                    <Flex
+                      bg={"#86efac"}
+                      w={125}
+                      h={50}
+                      gap={2}
+                      borderRadius={"15"}
+                      justifyContent={"center"}
+                      alignItems={"center"}
+                      color={"black"}
+                      fontSize={"2xl"}>
+                      <CheckIcon /> Resolve
+                    </Flex>
+                    <Text as={"b"} fontSize={"xl"} color={"black"}>
+                      Resolving a <span style={{ color: "blue" }}>FOUND</span>{" "}
+                      Item
+                    </Text>
+                    <Text fontSize={"lg"} color={"black"}>
+                      +5🍪
+                    </Text>
+                  </Flex>
+
+                  {/* Cookies for resolving a lost Item */}
+                  <Flex
+                    bg={"white"}
+                    boxShadow={"7px 7px 14px grey"}
+                    borderRadius={"xl"}
+                    p={5}
+                    gap={2}
+                    alignItems={"center"}
+                    justifyContent={"center"}
+                    flexDir={"column"}>
+                    <Flex
+                      bg={"#86efac"}
+                      w={125}
+                      h={50}
+                      gap={2}
+                      borderRadius={"15"}
+                      justifyContent={"center"}
+                      alignItems={"center"}
+                      color={"black"}
+                      fontSize={"2xl"}>
+                      <CheckIcon /> Resolve
+                    </Flex>
+                    <Text as={"b"} fontSize={"xl"} color={"black"}>
+                      Resolving a <span style={{ color: "red" }}>LOST</span>{" "}
+                      Item
+                    </Text>
+                    <Text fontSize={"lg"} color={"black"}>
+                      +2🍪
+                    </Text>
+                  </Flex>
+                </Flex>
+                <Flex justifyContent={"center"} alignItems={"center"} gap={5}>
+                  <Text as={"b"} fontSize={"xl"}>
+                    The more cookies you have, the higher you rank! 🏆
+                  </Text>
+                </Flex>
+
+                <Button
+                  ref={btnRef}
+                  colorScheme="blue"
+                  onClick={() => setShowCookieInfo(false)}>
+                  🡰 Back to Leaderboard
```

Comment:
I think this black box thing is supposed to be an emoji, but it isn't being recognized.